navigation3/navigation3-runtime/gradle.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2025 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# TODO Determine redirection group based on package and remove explicit config
+artifactRedirection.groupId=androidx.navigation3
```

Comment:
I think it should work without it because of `artifactRedirection.groupIdReplacement`